pkg/controller/utils/errorscrub.go

```diff
@@ -21,6 +22,7 @@ func ErrorScrub(err error) string {
 	}
 	s := newlineTabRE.ReplaceAllString(err.Error(), ", ")
 	s = awsRequestIDRE.ReplaceAllString(s, "")
+	s = awsNotAuthorized.ReplaceAllString(s, `$1 XXX $2 XXX $3 XXX $4`)
```

Comment:
You can get rid of the spaces by using curly braces:
```suggestion
	s = awsNotAuthorized.ReplaceAllString(s, `${1}XXX${2}XXX${3}XXX${4}`)
```

Reply:
Fantastic